nucypher/cli/painting.py

```diff
@@ -494,3 +499,77 @@ def paint_deployment_delay(emitter, delay: int = 3) -> None:
         time.sleep(1)
+
+
+def paint_stakers(emitter, stakers: List[str], agent) -> None:
+    current_period = agent.get_current_period()
+    emitter.echo(f"\nCurrent period: {current_period}")
+    emitter.echo("\n| Stakers |\n")
+    emitter.echo(f"{'Checksum address':42}  Staker information")
+    emitter.echo('=' * (42 + 2 + 53))
+
+    for staker in stakers:
+        nickname, pairs = nickname_from_seed(staker)
+        symbols = f"{pairs[0][1]}  {pairs[1][1]}"
+        emitter.echo(f"{staker}  {'Nickname:':10} {nickname} {symbols}")
+        tab = " " * len(staker)
+
+        stake = agent.owned_tokens(staker)
+        last_confirmed_period = agent.get_last_active_period(staker)
+        worker = agent.get_worker_from_staker(staker)
+
+        missing_confirmations = current_period - last_confirmed_period
+        stake_in_nu = round(NU.from_nunits(stake), 2)
+        locked_tokens = round(NU.from_nunits(agent.get_locked_tokens(staker)), 2)
+
+        emitter.echo(f"{tab}  {'Stake:':10} {stake_in_nu}  (Locked: {locked_tokens})")
+        emitter.echo(f"{tab}  {'Activity:':10} ", nl=False)
+        if missing_confirmations == -1:
+            emitter.echo(f"Next period confirmed (#{last_confirmed_period})", color='green')
+        elif missing_confirmations == 0:
+            emitter.echo(f"Current period confirmed (#{last_confirmed_period}). "
+                         f"Pending confirmation of next period.", color='yellow')
+        elif missing_confirmations == current_period:
+            emitter.echo(f"Never confirmed activity", color='red')
+        else:
+            emitter.echo(f"Missing {missing_confirmations} confirmations "
+                         f"(last time for period #{last_confirmed_period})", color='red')
+
+        emitter.echo(f"{tab}  {'Worker:':10} ", nl=False)
+        if worker == BlockchainInterface.NULL_ADDRESS:
+            emitter.echo(f"Worker not set\n", color='red')
+        else:
+            emitter.echo(f"{worker}\n")
+
+
+def paint_locked_tokens_status(emitter, agent, periods) -> None:
+
+    MAX_ROWS = 30
+    period_range = list(range(1, periods + 1))
+    token_counter = Counter({day: agent.get_all_locked_tokens(day) for day in period_range})
+
+    width = 60  # Adjust to desired width
+    longest_key = max(len(str(key)) for key in token_counter)
+    graph_width = width - longest_key - 2
+    widest = token_counter.most_common(1)[0][1]
+    scale = graph_width / float(widest)
+
+    bucket_size = periods // MAX_ROWS if periods > MAX_ROWS else 1
+
+    emitter.echo(f"\n| Locked Tokens for next {periods} periods |\n")
+
+    buckets = [period_range[i:i + bucket_size] for i in range(0, len(period_range), bucket_size)]
+
+    for bucket in buckets:
+        bucket_start = bucket[0]
+        bucket_end = bucket[-1]
+
+        bucket_max = max([token_counter[period] for period in bucket])
+        bucket_min = min([token_counter[period] for period in bucket])
```

Comment:
Visuals showing min, max and total tokens locked for each set of periods are great. Maybe a note explaining what `min` and `max` mean wouldn't go amiss?